timesketch/frontend-ng/src/components/Explore/TimelineChip.vue

```diff
@@ -371,6 +393,18 @@ export default {
       if (this.timelineStatus === 'processing') return 'mdi-circle-slice-7'
       return 'mdi-alert-circle-outline'
     },
+    timelineFailed() {
+      return this.timelineStatus === 'fail';
+    },
+    timelineChipColor() {
+      if (this.timelineStatus === 'ready' && !this.isSelected) {
+        return 'grey'
```

Comment:
To fix #2554 as well, could you keep the timelineChipColor but instead of turning it grey if it is inactive lower transparency/saturation?

Reply:
I changed it to keep the color. It has some transparency when disabled.